crates/sound/Cargo.toml

```diff
@@ -1,35 +1,37 @@
 [package]
 name = "vhost-user-sound"
 version = "0.1.0"
-authors = ["Stefano Garzarella <sgarzare@redhat.com>"]
+authors = ["Stefano Garzarella <sgarzare@redhat.com>", "Manos Pitsidianakis <manos.pitsidianakis@linaro.org>"]
 description = "A virtio-sound device using the vhost-user protocol."
 repository = "https://github.com/rust-vmm/vhost-device"
 readme = "README.md"
-keywords = ["vhost", "sound", "virtio-sound"]
+keywords = ["vhost", "sound", "virtio-sound", "virtio-snd", "vhost-user", "virtio"]
```

Comment:
there is a limit of 5 keywords when publishing to crates.io iirc

Reply:
fixed!